pages/landing/patcher/_features.html

```diff
@@ -3,8 +3,8 @@ <h2>Patcher Features</h2>
   {% for feature in page.features %}
     <div class="col-xs-6 col-md-4" style="margin-bottom:20px;">
       <div class="row" style="display: flex; flex-wrap: wrap;">
-        <div class="col-xs-12 col-sm-4 hidden-print" style="padding-right: 0; padding-left: 0;">
-          <img src="{{ feature.icon }}" alt="{{ feature.title }}" style="max-width: 105px;" />
+        <div class="col-xs-12 col-sm-3 hidden-print" style="padding-right: 5px;">
+          {{ feature.icon }}
         </div>
         <div class="col-xs-12 col-sm-8" style="line-height: 1.4;">
           <h4 class="h6 text-bold" style="margin: 5px 0 5px 0">{{ feature.title }}</h4>
```

Comment:
```suggestion
          <h4 class="h6 text-bold" style="margin: 0">{{ feature.title }}</h4>
```

If you get rid of the margin for each feature description, the headings for each feature will align with the top of the icons better too.